bench/abc/optimized/mvcApi.ll

```diff
@@ -241,13 +241,19 @@ Mvc_CoverMakeEmpty.exit:                          ; preds = %.lr.ph.i, %1
   %24 = zext nneg i32 %8 to i64
   %25 = getelementptr inbounds nuw i32, ptr %23, i64 %24
   store i32 %22, ptr %25, align 4, !tbaa !23
-  %26 = shl i32 %7, 2
-  %27 = and i32 %26, 67108860
-  %28 = zext nneg i32 %27 to i64
-  tail call void @llvm.memset.p0.i64(ptr nonnull align 4 %23, i8 -1, i64 %28, i1 false), !tbaa !23
-  br label %.loopexit
-
-.loopexit:                                        ; preds = %.lr.ph.preheader, %14, %9
+  %.027 = add nsw i32 %8, -1
+  br label %.lr.ph
+
+.lr.ph: ; preds = %.lr.ph.preheader, %.lr.ph
+  %.028 = phi i32 [ %.0, %.lr.ph ], [ %.027, %.lr.ph.preheader ]
+  %26 = zext nneg i32 %.028 to i64
+  %27 = getelementptr inbounds nuw i32, ptr %23, i64 %26
+  store i32 -1, ptr %27, align 4, !tbaa !23
+  %.0 = add nsw i32 %.028, -1
+  %28 = icmp ult i32 %.0, 16777215
+  br i1 %28, label %.lr.ph, label %.loopexit, !llvm.loop !29
+
+.loopexit:                                        ; preds = %.lr.ph, %14, %9
```

Comment:
Not a regression: the new version expands the `call void @llvm.memset.p0.i64`, which is faster but with more IR lines.